environment.yml

```
@@ -0,0 +1,10 @@
+name: gmtforgeodesy
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - gmt=6.3.0
+  - jupyterlab=3.4.0
+  - python=3.9
+  # Optional dependencies
+  - jupyter-book=0.12.3
```

Just a note that jupyter-book released 0.13.0 yesterday (https://github.com/executablebooks/jupyter-book/releases/tag/v0.13). They've migrated from sphinx-panels to sphinx-design, so you would need to change the cards in `intro.md`. Up to you of course whether to do the migration.

```suggestion
  - jupyter-book=0.13.0
```

---

Thanks, I will leave this comment open for now and address it as a separate issue after the initial PR due to time restrictions.